python/lsst/ip/diffim/getTemplate.py

```diff
@@ -522,8 +495,23 @@ class GetMultiTractCoaddTemplateConnections(pipeBase.PipelineTaskConnections,
     )
 
 
-class GetMultiTractCoaddTemplateConfig(pipeBase.PipelineTaskConfig, GetCoaddAsTemplateConfig,
-                                       pipelineConnections=GetMultiTractCoaddTemplateConnections):
+class GetTemplateConfig(pipeBase.PipelineTaskConfig,
+                        pipelineConnections=GetTemplateConnections):
+    templateBorderSize = pexConfig.Field(
+        dtype=int,
+        default=10,
+        doc="Number of pixels to grow the requested template image to account for warping"
+    )
+    coaddName = pexConfig.Field(
```

Comment:
You can delete the configs `coaddName` and `warpType` now.   In Gen3 they are templates in the connection names, not configs.